openedx/core/djangoapps/announcements/templates/announcements/announcements_list.html

```diff
@@ -0,0 +1,19 @@
+{% block title %}{{_("Manage Announcements")}}{% endblock %}
+
+<div class="announcements-list">
+  {% for announcement in announcement_list %}
+    <a href="{% url 'announcements:announcements_edit' pk=announcement.pk %}">
+      <div class="announcement-item {% if announcement.active %}announcement-active{% endif %}">
```

Comment:
The "active" class state with styling is too implicit, and not accessible. 
Better to provide a table that labels each announcement as "active" or "inactive".

Also need to allow for pagination here.